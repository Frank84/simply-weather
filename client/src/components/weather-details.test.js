import React from 'react';
import WeatherDetails from './search-input';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  shallow(<WeatherDetails />);
});