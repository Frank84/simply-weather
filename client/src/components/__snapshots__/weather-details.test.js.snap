// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <Search
    autoFocus={true}
    icon="search"
    input="text"
    loading={false}
    minCharacters={1}
    noResultsMessage="No results found."
    onResultSelect={[Function]}
    onSearchChange={[Function]}
    placeholder="Search for a city"
    resultRenderer={[Function]}
    results={Array []}
    showNoResults={true}
    value=""
  />
  <Icon
    as="i"
    className=" search-icon"
    name="search"
    onClick={[Function]}
  />
</form>
`;
